components/Bubbles/BubblesGame.js

```diff
@@ -277,33 +306,22 @@ class BubblesGame extends React.Component {
     clearInterval(this.bubbleFountainInterval);
   }
 
-  homeBtn = () => {
-    this.props.navigator.replace({
-      id: 'Main',
-    });
-  }
-
   render () {
     return (
       <Image source={require('../../media/backgrounds/Game_7_Background_1280.png')} style={styles.backgroundImage}>
-          <View style={styles.topBar}>
-            <TouchableOpacity style={styles.button} onPress={this.buttonPress}>
-            </TouchableOpacity>
-          </View>
           <View style={styles.gameWorld}>
-
             <AnimatedSprite
               character={lever}
               characterUID={this.characterUIDs.lever}
               animationFrameIndex={[0]}
               loopAnimation={false}
               coordinates={{
-                top: 100 * this.scale.screenHeight,
-                left: 1077 * this.scale.screenWidth}}
+                top: LEVER_LOCATION.top,
+                left: LEVER_LOCATION.left}}
               size={{
```

Comment:
instead of Math.floor(214 * this.scale.image) use Math.floor(lever.size.width * this.scale.image). Technically I think we can ignore Math.floor and it gets done automatically.